### PR TITLE
📜 Curlies for english 

### PR DESCRIPTION
**Pull request by SabinaChita**
@Felienne: 

I have cleaned up the curlies from level 18 up to level 14 (all tabs including quiz). 

At level 14 by the "is" tab I am not sure what part of the code has to be where. See screenshots, the "localhost url" is what I made, the other one is how it is looks in reality.

whatimade
whatisnow

### CODE REVIEW

File: content/adventures/en.yaml [hunk 1]
```diff
@@ -1428,7 +1428,7 @@ adventures:
                 story_text: |
                     Now we are going to change indentation a little bit. Every time that we need an indentation, we need `:` at the line before the indentation.
 
-                    In this level you can also use a new command: `{elif}`. `{elif}` is short for {else} {if} and you need it when you want to make 3 (or more!) options.
+                    In this level you can also use a new command: `{elif}`. `{elif}` is short for `{else} {if}` and you need it when you want to make 3 (or more!) options.
```
Comment (Felienne) [inline content/adventures/en.yaml:1431]: Here too, are there untranslated languages with the same issue? In that case updating them there too would be lovely!

**Comment by Felienne** [comment]
> @Felienne:
> 
> I have cleaned up the curlies from level 18 up to level 14 (all tabs including quiz).
> 
> At level 14 by the "is" tab I am not sure what part of the code has to be where. See screenshots, the "localhost url" is what I made, the other one is how it is looks in reality.
> 
> whatimade whatisnow

Your version looks perfect!

File: content/adventures/en.yaml [hunk 2]
```diff
@@ -2435,11 +2442,6 @@ adventures:
                     {if} name != 'Hedy'
                         {print} 'You are not Hedy'
                     ```
-
-                    {if} age < 13
```
Comment (Felienne) [inline content/adventures/en.yaml:2439]: Yeah this part was just there in error I think!

File: content/adventures/en.yaml (hunk 1 above)
Comment (SabinaChita) [reply content/adventures/en.yaml:1431]: yes of course, I will have a look.

File: content/adventures/en.yaml (hunk 2 above)
Comment (SabinaChita) [reply content/adventures/en.yaml:2439]: Great, thank you! I'll get back to work!

**Comment by Felienne** [comment]
When #5174 is merged, this needs to go in to remove the risk of conflicts!

**Review by Felienne — APPROVED** [review]

**Comment by Felienne** [comment]
Hi @SabinaChita! We have merged this, if you want to continue from level 13 downwards, do make a new PR!

**Pull request merged**